/softhand_grasp_node:
   ros__parameters:
      
      'bend_angle_array_point0': [0.0, 41.0, 270.0, 35.0, 0.0]  #in degrees, range:[0, 300]
      'wave_angle_array_point0': [40.0, 4.35, 2.8, 32.0, 10.0]   # range:[-150, 150]
      'bend_angle_array_point1': [130.0, 175.0, 155.0, 150.0, 140.0]    
      'bend_angle_array_point_shaking': [65.0, 90.0, 220.0, 80.0, 70.0]  
      #'bend_angle_array_point1': [175.0, 160.0, 165.0, 150.0, 150.0]    
      'wave_angle_array_point1': [40.0, 4.35, 2.8, 32.0, 10.0]    

      
/softhand_setspeed_clientnode:
   ros__parameters:
      'bend_speed_array': [22.0, 22.0, 22.0, 22.0, 22.0]  #in degrees/sec, range:[0-684]
      'wave_speed_array': [15.0, 15.0, 15.0, 15.0, 15.0]  
